packages/opentelemetry-propagator-b3/src/B3MultiPropagator.ts

```diff
@@ -42,7 +42,7 @@ function isValidSampledValue(sampled: TraceFlags | undefined): boolean {
   return sampled === TraceFlags.SAMPLED || sampled === TraceFlags.NONE;
 }
 
-export function parseHeader(header: unknown) {
+function parseHeader(header: unknown) {
```

Comment:
I removed `export` by intuition. Let me know if I did wrong here.

Reply:
If the tests pass it's probably fine. It might have been exported for those